src/Umbraco.Web.BackOffice/Controllers/MemberController.cs

```diff
@@ -143,7 +149,8 @@ public class MemberController : ContentControllerBase
               jsonSerializer,
               passwordChanger,
               scopeProvider,
-              StaticServiceProvider.Instance.GetRequiredService<ITwoFactorLoginService>())
+              StaticServiceProvider.Instance.GetRequiredService<ITwoFactorLoginService>(),
+              null)
```

Comment:
This should get the options/settings using the `StaticServiceProvider.Instance` as fallback:
```suggestion
              StaticServiceProvider.Instance.GetRequiredService<IOptions<SecuritySettings>>())
```

Reply:
Thanks, change made.